doc/benchmarking_tutorial.rst

```diff
@@ -0,0 +1,97 @@
+Benchmarking tutorial
+=====================
+The [benchmarking package](https://github.com/ros-planning/moveit/tree/kinetic-devel/moveit_ros/benchmarks) provides methods to benchmark motion planning algorithms and aggregate/plot statistics.
+The example below demonstrates how the benchamrking can be run for a Fanuc M10ia.
+
+Example
+-------
+
+An example is provided in the examples folder. The launch file requires a moveit config package 
+for the Fanuc M10ia available from [here](https://github.com/ros-planning/moveit_resources).
+
+To run:
+
+1. Launch the Fanuc M10ia demo.launch: 
+    ```
+    roslaunch moveit_resources demo.launch db:=true
+    ```
+2. Connect to the database in the motion planning context tab
+3. Save a scene and associated query for the Fanuc M10ia. Name the scene Kitchen1 and the 
+   query Pick1.
+4. Also save a start state for the robot and name it Start1.
+5. The config file demo1.yaml refers to the scenes, queries and start states used for benchmarking. Modify them appropriately.
+6. Now, bring down your previous launch file and then run the benchmarks: 
+    ```
+    roslaunch moveit_ros_benchmarks demo_fanuc.launch
+    ```
+
+Viewing Results
+-------
+The benchmarks are executed and many interesting parameters are aggregated and written to a logfile.  A script (moveit_benchmark_statistics.py) is supplied to parse this data and plot the statistics.
+Run:
+```
+python moveit_benchmark_statistics.py <path_of_logfile>
+```
+To generate a PDF of plots:
+```
+python -p <plot_filename> moveit_benchmark_statistics.py <path_of_logfile>
+```
+
```

Comment:
I'd add a sentence here like this: "Alternatively, upload the benchmark.db file generated by moveit_benchmark_statistics.py to http://plannerarena.org and interactively visualize the results." (I assume that moveit_benchmark_statistics.py also generates a SQLite3 database like ompl_benchmark_statistics.py. Both scripts should be (nearly) identical.) The hard-coded scripts in moveit_benchmark_statistics.py are somewhat limited. I like the look of the ggplot-generated plots on planner arena.org better. Plus, you can download these plots, tweak them & make them "camera-ready" in R.